docs/releases/5.1.2.txt

```diff
@@ -9,4 +9,6 @@ Django 5.1.2 fixes several bugs in 5.1.1.
 Bugfixes
 ========
 
-* ...
+* Fixed a regression in Django 5.1 that caused a crash of ``.filter(...)`` when 
+  ``Concat`` is used in combination with ``trigram_similar`` due to using 
+  double pipes without wrapping it in parenthesis (:ticket:`35732`).
```

Comment:
```suggestion
* Fixed a regression in Django 5.1 that caused a crash when using the
  PostgreSQL lookup :lookup:`trigram_similar` on output fields from ``Concat``
  (:ticket:`35732`).
```
Maybe?
The release note should just detail what got fixed rather than how it was fixed :+1: